packages/foam-core/src/utils.ts

```diff
@@ -2,7 +2,11 @@ import path from 'path';
 import crypto from 'crypto';
 import { titleCase } from 'title-case';
 import GithubSlugger from 'github-slugger';
-import { URI, ID } from 'types';
+import { URI, ID } from './types';
+
+export function isNotNull<T>(p: T | null): p is T {
+  return p != null;
```

Comment:
What does p stand for? 

nit: consider adding a comment or renaming to be more clear?

Reply:
good point, should just be `value` - no idea what I was thinking about when I went for `p` (I am sure it was very clever at the time ;) )